snap/snapcraft.yaml

```diff
@@ -0,0 +1,33 @@
+name: flamenco
+base: core22
+version: '1.0.0'
```

Comment:
issue: This project is still in development. I would like to apply a `0.X.Y` schema for this state to adhere to [semantic versioning](https://semver.org/).

note: additionally I would like to get the version from the project itself
- either via `flamenco --version`
- or via parsing the <Version> field from the `.csproj` file

This requires setting `adopt-info: metadata` and adding `craftctl set version="${FLAMENCO_VERSION}+git.$(git rev-parse --short HEAD)"` somewhere in the snapcraft build process.

Reply:
We would need to build flamenco any way to build the snap, so there should be no overhead, but I would like it more if `flamenco --version` would output a short form hash

Reply:
That's right, I could set the version with `flamenco --version` in `override-prime`, for example, but:
```
:: + craftctl set version=1.0.0+d360539e2b7b37e313de016660b0a90f41307194                                                                                              
Extracting and updating metadata...                                                                                                                                   
error setting version: ensure this value has at most 32 characters
```

So yes, we either get `--version` to output a short hash or I can do that manually with some some bash magic during `craftctl set version`